client/components/diff-window.js

```diff
@@ -320,7 +320,7 @@ const DiffWindow = (props) => {
   const compare = () => {
 
     const hasVisibleChanges = hasContentChanges(before, props.value, props.type);
-    let { removed, added } = changes.removed;
+    let { removed, added } = changes;
```

Comment:
I tested my changes without your changes and everything seemed to work fine after fix in line 51. 

When I tested with your changes line 323 is causing issues in couple of places, above change seems to have fixed the scenarios I was testing. I don't think I have tested the scenarios related to your bug. 

Could you please check? - @parvinba

Reply:
looks ok to me...